skrifa/src/outline/autohint/latin/edges.rs

```diff
@@ -21,10 +21,11 @@ pub(crate) fn compute_edges(
     metrics: &ScaledAxisMetrics,
     top_to_bottom_hinting: bool,
     y_scale: i32,
+    group: ScriptGroup,
 ) {
     axis.edges.clear();
     let scale = metrics.scale;
-    let top_to_bottom_hinting = if axis.dim == Axis::HORIZONTAL {
+    let top_to_bottom_hinting = if axis.dim == Axis::HORIZONTAL || group != ScriptGroup::Default {
```

Comment:
horizontal implies not top-to-bottom makes sense. The group limitation somewhat less so. I suggest a comment.

Reply:
I'm probably being overly defensive here but the CJK code always passes constant 0 where this value is expected. I added a comment with a link to FT code